canonical-web-and-design/snapcraft.io
# Add discourse-docs extension

# Done
Added discourse-docs extension, to make snapcraft documentation available under the `/docs` prefix.

# QA
Navigate to `/docs` and make sure the documentation works and everything renders properly.
